client/src/client.ts

```diff
@@ -116,6 +117,19 @@ function registerNotificationHandlers(client: lsp.LanguageClient) {
         }),
     );
   });
+
+  const d2 = client.onNotification(SuggestStrictMode, async (params: SuggestStrictModeParams) => {
+    const selection = await vscode.window.showInformationMessage(
+        params.message,
+        'Open tsconfig.json',
+    );
+    if (selection) {
```

Comment:
```suggestion
    const openTsConfig = 'Open tsconfig.json';
    const selection = await vscode.window.showInformationMessage(
        params.message,
        openTsConfig,
    );
    if (selection === openTsConfig) {
```
While I know the API for `showInformationMessage`, I think this might be a bit more clear. WDYT?